cds_ils/config.py

```diff
@@ -420,6 +420,8 @@ def _parse_env_bool(var_name, default=None):
 
 #: The maximum duration of a loan request
 ILS_CIRCULATION_LOAN_REQUEST_DURATION_DAYS = 120
+#: Loan Request Offset Value
+ILS_CIRCULATION_LOAD_REQUEST_OFFSET = 2
```

Comment:
```suggestion
ILS_CIRCULATION_LOAN_REQUEST_OFFSET = 2
```
Same comment here related to the docstring